_posts/2016-03-17-the-need-for-fit.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "The need for fit"
+date:   2016-03-17 11:58:26
+author: Michael Peyton Jones
+categories:
+---
+
+Personal fit with project the is incredibly important in startup founders. 
```

Comment:
We should look at what people like Paul Graham, Peter Thiel, and Matt Clifford have to say about this. Especially Matt Clifford as EF has more and more been emphasising that they want people with prior deep technical skills that they can apply to a project, rather than just general devs to make web app startups.